### DISCUSSION TITLE
第4节课作业(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#229] [作业]
请提交作业的链接 Please submit a link to the assignment.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/678212137

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/4-XTuner-78ed9b40fd504a88adddd9c81ac63c5b

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/679314163
作业与笔记是同一个连接

**Comment by jayjayhust** [comment]
https://zhuanlan.zhihu.com/p/679609413

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/679621953

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679707416

**Comment by YongXie66** [comment]
https://zhuanlan.zhihu.com/p/679766841

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/4
